sklearn/utils/tests/test_random.py

```diff
@@ -180,3 +184,28 @@ def test_random_choice_csc_errors():
     class_probabilities = [np.array([0.5, 0.6]), np.array([0.6, 0.1, 0.3])]
     assert_raises(ValueError, random_choice_csc, 4, classes,
                   class_probabilities, 1)
+
+
+@pytest.mark.parametrize("low,high,base", [
+    (-4, 0, 10),
+    (-6, 1, np.exp(1)),
+    (-8, 4, 2),
+])
+def test_log_uniform(low, high, base):
+    rv = loguniform(low, high, base=base)
+    rvs = rv.rvs(size=100)
+    assert (base**low <= rvs).all() and (rvs <= base**high).all()
+
+    y = loguniform(low, high)
+    rvs = y.rvs(size=100)
+    assert (10**low <= rvs).all() and (rvs <= 10**high).all()
+    assert isinstance(y.rvs(), np.ndarray)
+
+
+def test_log_uniform_param_sampler(n_iter=10):
```

Comment:
If we've tested ParameterSampler with a custom class with rvs, this tests nothing new.

Reply:
There weren't any tests for ParameterSampler with a custom class. I added one.